docs/spock_functions/index.md

```
@@ -20,9 +20,9 @@ The following user functions are available via the Spock extension:
 | [spock.repset_add_all_tables](functions/spock_repset_add_all_tables.md) | Add all existing table(s) to the replication set.
 | [spock.repset_remove_table](functions/spock_repset_remove_table.md) | Remove table from replication set.
 | spock.repset_show_table |
-| [repset_add_seq](functions/spock_repset_add_seq.md) | Deprecated; Adds a sequence to a replication set.
-| [repset_add_all_seqs](functions/spock_repset_add_all_seqs.md) | Deprecated; Adds all sequences from the specified schemas to a replication set.
-| [repset_remove_seq](functions/spock_repset_remove_seq.md) | Deprecated; Remove a sequence from a replication set.
+| [spock.repset_add_seq](functions/spock_repset_add_seq.md) | Adds a sequence to a replication set.
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)